actions/workflows/auth_chatops.yaml

```diff
@@ -1,44 +1,42 @@
 ---
-version: '2.0'
-duo.auth_chatops:
-  description: "Only send an announcement if successfully auth'ed with Duo"
-
-  input:
-    - uuid
-    - user
-    - channel
-    - hint
-
-  output:
-    auth: <% $.auth %>
-    message: <% $.message %>
-
-  tasks:
-    duo_auth:
-      # [235, 26]
-      action: duo.auth_auth
-
-      input:
-        username: <% $.user %>
-        pushinfo: {"Source": "ChatOps", "Action": "<% $.hint %>"}
-
-      on-success:
-        - announce_success
-
-      on-error:
-        - post_failure
-
-    announce_success:
-      # [105, 128]
-      action: duo.announce_success
-      input:
-        uuid: <% $.uuid %>
-
-    post_failure:
-      # [365, 128]
-      action: chatops.post_message
-      input:
-        channel: <% $.channel %>
-        whisper: false
-        message: "*auth failure:* two-factor auth attempt failed.{~}"
-        user: <% $.user %>
+version: '1.0'
+description: Only send an announcement if successfully auth'ed with Duo
+
+input:
+  - uuid
+  - user
+  - channel
+  - hint
+
+output:
+  - auth: "{{ ctx().auth }}"
+  - message: "{{ ctx().message }}"
+
+tasks:
+  duo_auth:
```

Comment:
The comments with the # of the placement of the boxes has been lost in the translation - don't know if its preferable to keep them - it's obviously doesn't stop it working - so not sure if worth keeping or not.  So leaving as an observation but not blocking merge.

Reply:
The  # comments are used by the web UI for Orquesta workflows (in the EWC workflow designer they definitely affect how it shows the boxes), however as the placement of them in an action-chain might be different to the orquesta designer then it probably doesn't matter that they are lost, as the two views might differ anyway.  So the numbers might be the wrong values, so in hind-sight probably best to remove.